third_party/icu/source/common/uassert.h

```diff
@@ -32,9 +32,7 @@
 #include "starboard/configuration.h"
 #endif  // defined(STARBOARD)
 #if U_DEBUG
-#if !defined(STARBOARD)
-#   include <assert.h>
```

Comment:
Leave this line unchanged?
https://chromium.googlesource.com/chromium/deps/icu/+/c9630bce/source/common/uassert.h#32

Reply:
Done